server/controllers/index.js

```diff
@@ -1,10 +1,10 @@
 module.exports = {
-    patterns: {
-        index: (ctx) => ctx.render('patterns/index', {}),
-        typography: (ctx) => ctx.render('patterns/typography', {}),
-        grids: (ctx) => ctx.render('patterns/grids', {}),
-        palette: (ctx) => ctx.render('patterns/palette', {}),
-        icons: (ctx) => ctx.render('patterns/icons', {})
-    },
-    healthcheck: (ctx) => ctx.body = 'ok'
+  patterns: {
+    index: (ctx) => ctx.render('patterns/index', {}),
+    typography: (ctx) => ctx.render('patterns/typography', {}),
+    grids: (ctx) => ctx.render('patterns/grids', {}),
+    palette: (ctx) => ctx.render('patterns/palette', {}),
+    icons: (ctx) => ctx.render('patterns/icons', {})
+  },
+  healthcheck: (ctx) => ctx.body = 'ok'
```

Comment:
The fact that es6 allows you to do this to create more concise, functional code, I'd be tempted to turn it off.